tpgtools/overrides/cloudbuildv2/beta/tpgtools_product.yaml

```diff
@@ -0,0 +1,5 @@
+## product level overrides
+
+- type: PRODUCT_DOCS_SECTION
+  details:
+    docssection: Cloud Build
```

Comment:
We map our docs sections to the endpoints being hit, rather than a subjective grouping of what a product or service is. We'll want to use the natural name "Cloud Build v2", or override to something like "Cloud Build (v2 API)" or similar. See Cloud Functions and Cloud Run  & their v2s for reference.